docker-compose/config/worker-ubuntu22-04.jsonnet

```diff
@@ -31,6 +80,11 @@ local common = import 'common.libsonnet';
       },
     }],
   }],
+  filePool: {
+    blockDevice: {
+      devicePath: '/worker/filepool',
```

Comment:
I think this won’t work. You still need to specify a size. You can typically set this to the quota size * concurrency.

Reply:
Oh wait. It could work, if that path is a device. But is it realistic to use an actual raw block device here? For a simple Docker compose setup, I guess people would want to make it file backed.

Reply:
yeah, it doesn't work unless it's a file configuration or a raw block device.

However @moroten set up a file configuration instead!
https://github.com/buildbarn/bb-deployments/pull/84